minifiers/uglifyjs.js

```diff
@@ -77,7 +77,8 @@ module.exports = function uglify(options, done) {
     filename: filename
   }, {
     [map]: {
-      content: results.map
+      content: results.map,
+      fingerprint: fingerprint.id
```

Comment:
yeah.. not feeling strong about bringing this back. Although it's not doing anything atm, it might be useful for future work like the CLI directly running dry-runs of builds and assessing the expected output. Although technically that should still go through `bffs` normalization as well.

Reply:
agreed. I think we can clean up this entire interface and how we interop with bffs in the future so we dont have so much transforming and intermediary formats.